fix: prevent number and units from being wrapped

pixelspark opened the pull request:
This PR prevents statistics with units (such as `1:23 min/km`) from being wrapped over two lines. This looks much nicer imo.

Before:

<img width="325" alt="image" src="https://github.com/jovandeginste/workout-tracker/assets/181337/6c82a3b8-7f6c-4abf-a7c5-3d02d66d391e">

After:

<img width="344" alt="image" src="https://github.com/jovandeginste/workout-tracker/assets/181337/c764d9cb-750e-4814-9079-49ff950c2397">

I applied the class `whitespace-nowrap` to all places I could find, might have missed some though.

jovandeginste commented:
Can you check if the views remain okay when resizing the viewport? I do this by switching to "mobile view" in firefox (`ctrl+shift+m`), and then resizing the viewport there...

pixelspark commented:
Looks okay. 

<img width="369" alt="image" src="https://github.com/jovandeginste/workout-tracker/assets/181337/d8a681c5-739a-4e4e-8a5a-614f9dc79336">

<img width="339" alt="image" src="https://github.com/jovandeginste/workout-tracker/assets/181337/a7a1b95d-3225-4be2-9e53-b40ad684c90e">

pixelspark commented:
Also, should add 'font-mono' for all the others, just a sec!

jovandeginste commented:
FYI, the test fails because `DOCTYPE` is now capitalized in `views/workouts/workouts_list.html`; no idea why :)

jovandeginste commented:
ah, and icons for the new workout types

pixelspark commented:
OK, fixed the doctype and added `font-mono` to all places where I also applied `whitespace-nowrap`.

<img width="363" alt="image" src="https://github.com/jovandeginste/workout-tracker/assets/181337/dd62f35d-9433-4061-9ad6-c04f123bb92e">

What is the issue with the new icons? I added them in the previous PR and they appear to work fine for me?

jovandeginste commented:
![image](https://github.com/jovandeginste/workout-tracker/assets/3170771/905fd3b5-f1a6-4f49-a713-6fceb4b1bce2)

pixelspark commented:
Weird, somehow that change didn't make it! (I was very proud to have found the "person-drowning" icon for kayaking :-))
